bundles/af-core/src/test/resources/form/termsandconditions/exporter-termsandconditionsNoWrapData.json

```diff
@@ -9,7 +9,8 @@
       "dorExcludeDescription": false
     },
     "fd:path": "/content/termsandconditionsNoWrapData",
-    "fd:tnc": true
+    "fd:tnc": true,
+    "showAsPopup": true
```

Comment:
The exported key should have the fd: prefix. My previous comment referred to the property saved in JCR.